public/react/SettingsSidebar/components/IssueSource/components/StatusSelect/StatusSelect.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC, useId } from "react";
+import Select from "@atlaskit/select";
+import { Label } from "@atlaskit/form";
+import { StatusSelectProps } from "./types";
+
+const StatusSelect: FC<StatusSelectProps> = ({ label, placeholder, options, value, onChange }) => {
```

Comment:
I implemented this component as a dummy component where the parent is responsible for passing the data to it directly but thinking about it now, this component now looks like a Generic select component instead of a Status Select component.

Do you think this component will become more useful if its a stateful Status Select component and let it take care of defining the statuses state, handlers, etc..?

Reply:
I removed this component, probably not worth encapsulating the logic in this `StatusSelect` component if its not going to be reused somewhere else. Especially that its going to be tied to just `excludedStatuses`